addons/auth_totp/models/res_users.py

```diff
@@ -130,6 +130,14 @@ def totp_enable_wizard(self):
             'views': [(False, 'form')],
         }
 
+    @api.model
+    def _neutralize(self):
+        super()._neutralize()
+        user_ids = self.search([('totp_secret', '!=', False)])
+        user_ids.update({'totp_secret': False})
```

Comment:
Please avoid `update()`, and use `write()` or field assignment instead.
The method `update()` is not as safe as the other ones.